cosmwasm/utils.js

```diff
@@ -737,6 +762,15 @@ const encodeParameterChangeProposal = (options) => {
     };
 };
 
+const encodeMigrateContractProposal = (config, options, chainName) => {
+    const proposal = MigrateContractProposal.fromPartial(getMigrateContractParams(config, options, chainName));
+
+    return {
```

Comment:
Can we use the cosmwasm JS SDK to encode this instead of hardcoding?

Reply:
I'm already using cosmjs, or what do you mean?  If you mean using the `SigningCosmWasmClient` like we do with non-governance stuff, it doesn't support directly submitting a proposal, so I need to build the message and then broadcast it

Reply:
yeah, i meant for encoding the gov Msg below. Weird that it doesn't support it. Are we on the right version?